CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 - Added exception for unsupported types ([#1229](https://github.com/neptune-ai/neptune-client/pull/1229))
 - Change run status to Active / Inactive ([#1233](https://github.com/neptune-ai/neptune-client/pull/1233))
 - Package renamed from `neptune-client` to `neptune` ([#1225](https://github.com/neptune-ai/neptune-client/pull/1225))
+- Changed return values for run state ([#1253](https://github.com/neptune-ai/neptune-client/pull/1253))
```

Comment:
Let's change it to something about changing values for filtering runs table with the state. + Let's update the old one (Line 22) to something about returned values from the runs table state value.